src/scripts/main.js

```diff
@@ -1,8 +1,31 @@
 'use strict';
 
-const people = require('./lib/people');
+const people = require('./lib/people.json');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+console.log(people);
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  if (person.sex === 'm') {
+    person.sex = 'Male';
+  };
+
+  if (person.sex === 'f') {
+    person.sex = 'Female';
+  };
+
+  row.innerHTML = `
+    <th>${person.name}</th>
+    <th>${person.sex}</th>
+    <th>${person.born}</th>
+    <th>${person.died}</th>
+    <th>${person.died - person.born}</th>
+    <th>${Math.ceil(person.died / 100)}</th>
```

Comment:
Let's move `person.died - person.born` and `Math.ceil(person.died / 100)` to variables and use them